torch/_inductor/decomposition.py

```diff
@@ -325,6 +325,14 @@ def dequantize_per_tensor_tensor_decomp_impl(
 ) -> torch.Tensor:
     return (input.to(torch.float32) - zero_point) * scale
 
+@register_decomposition(aten._foreach_addcmul.Scalar)
+def _foreach_addcmul_scalar(self, left_tensors, right_tensors, scalar=1):
+    return aten._foreach_add.List(self, aten._foreach_mul.List(left_tensors, right_tensors), alpha=scalar)
```

Comment:
would this have wrong type promotion properties if some of the tensors are 0d? type promotion is not associative unfortunately

Reply:
Hmm let me play with this, this isn't strictly needed at this location, just thought it was easier than modifying the lowerings